libs/geoservices/internal/src/lib/pages/account/details/details.component.spec.ts

```diff
@@ -1,35 +1,32 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
 
-import { DetailsComponent } from './details.component';
 import { UIFormsModule } from '@tamu-gisc/ui-kits/ngx/forms';
-import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { EnvironmentModule, env } from '@tamu-gisc/common/ngx/environment';
-import { HttpClientTestingModule } from '@angular/common/http/testing';
-
 
+import { DetailsComponent } from './details.component';
 
 describe('DetailsComponent', () => {
   let component: DetailsComponent;
   let fixture: ComponentFixture<DetailsComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      imports: [FormsModule, ReactiveFormsModule, UIFormsModule, EnvironmentModule, HttpClientTestingModule],
+      imports: [ReactiveFormsModule, UIFormsModule, EnvironmentModule, HttpClientTestingModule],
       providers: [
         {
-          provide: env, 
-          useValue: { api_url : 'https://' }
-        }],
-      declarations: [ DetailsComponent ]
-    })
-
-    .compileComponents();
+          provide: env,
+          useValue: { api_url: 'https://' }
+        }
+      ],
+      declarations: [DetailsComponent]
+    }).compileComponents();
   }));
 
   beforeEach(() => {
     fixture = TestBed.createComponent(DetailsComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
```

Comment:
`detectChanges()` triggers Angular's change detection, including life cycle hooks. Without it, members like `ngOnInit` will not execute.

Avoid removing this statement.